### ISSUE TITLE
[BUG] houdini 20 spawned parameter windows weird location and size

### ISSUE THREAD

**Issue by NicTanghe** [leftwm/leftwm#1172]
### Describe the bug

When Houdini changed their method of handling parameter windows ects when it increased major versions, it broke on LEFTWM.

The windows spawnedt weird sizes and all over the place. (the windows are somewhat transparent.)
this issue does not seem to happen on GNOME.

### Expected behavior (optional)

The windows spawn in their expected location

### Steps to reproduce (optional)

Download Houdini apprentice.

In the node section on the bottom right, press tab and type box.
double-click the box in the node menu press tab and type pex.

select extrude in the nodes in the node tree.

Normally by now there should be 2 slightly gray tops of the transparent windows visible in weird locations

I have this in my config.ron :

( window_class: "Houdini FX", spawn_floating:  true),

### Relevant log output (optional)

```text
Below you can c some xprop stats.

Parameter window / Buggy spawned window:

❯ xprop
_NET_WM_DESKTOP(CARDINAL) = 2
WM_STATE(WM_STATE):
		window state: Normal
		icon window: <field not available>
_NET_WM_STATE(ATOM) =
_NET_WM_USER_TIME(CARDINAL) = 1404491
WM_TRANSIENT_FOR(WINDOW): window id # 0x4a00038
_NET_WM_ICON(CARDINAL) =
_NET_WM_ICON_NAME(UTF8_STRING) =
XdndAware(ATOM) = BITMAP
WM_NAME(STRING) =
_NET_WM_NAME(UTF8_STRING) = "Houdini FX"
_MOTIF_WM_HINTS(_MOTIF_WM_HINTS) = 0x2, 0x1, 0x0, 0x0, 0x0
_NET_WM_WINDOW_TYPE(ATOM) = _NET_WM_WINDOW_TYPE_UTILITY, _KDE_NET_WM_WINDOW_TYPE_OVERRIDE, _NET_WM_WINDOW_TYPE_NORMAL
_XEMBED_INFO(_XEMBED_INFO) = 0x0, 0x1
WM_CLIENT_LEADER(WINDOW): window id # 0x4a0000b
WM_HINTS(WM_HINTS):
		Client accepts input or input focus: True
		window id # of group leader: 0x4a0000b
WM_CLIENT_MACHINE(STRING) = "RHEL-FX-workstation-01"
_NET_WM_PID(CARDINAL) = 36884
_NET_WM_SYNC_REQUEST_COUNTER(CARDINAL) = 77594963
WM_CLASS(STRING) = "Houdini FX", "Houdini FX"
WM_PROTOCOLS(ATOM): protocols  WM_DELETE_WINDOW, WM_TAKE_FOCUS, _NET_WM_PING, _NET_WM_SYNC_REQUEST
WM_NORMAL_HINTS(WM_SIZE_HINTS):
		user specified location: 2257, 419
		user specified size: 252 by 36
		program specified minimum size: 252 by 36
		program specified maximum size: 252 by 32767
		window gravity: Static
```

main program / host window

```
❯ xprop
_NET_WM_DESKTOP(CARDINAL) = 0
WM_STATE(WM_STATE):
		window state: Normal
		icon window: <field not available>
_NET_WM_USER_TIME(CARDINAL) = 1402278
_NET_WM_STATE(ATOM) = _NET_WM_STATE_MAXIMIZED_HORZ, _NET_WM_STATE_MAXIMIZED_VERT
_NET_WM_ICON(CARDINAL) =
_NET_WM_ICON_NAME(UTF8_STRING) = "boat_orig_tweak_1.hiplc"
XdndAware(ATOM) = BITMAP
WM_NAME(STRING) = "/home/nanghe/CGI/Class/ChrisBruce/Holder/Class_2/boat_orig_tweak_1.hiplc - Houdini Indie Limited-Commercial 20.0.525 - Py3.10"
_NET_WM_NAME(UTF8_STRING) = "/home/nanghe/CGI/Class/ChrisBruce/Holder/Class_2/boat_orig_tweak_1.hiplc - Houdini Indie Limited-Commercial 20.0.525 - Py3.10"
_MOTIF_WM_HINTS(_MOTIF_WM_HINTS) = 0x3, 0x3e, 0x7e, 0x0, 0x0
_NET_WM_WINDOW_TYPE(ATOM) = _NET_WM_WINDOW_TYPE_NORMAL
_XEMBED_INFO(_XEMBED_INFO) = 0x0, 0x1
WM_CLIENT_LEADER(WINDOW): window id # 0x4a0000b
WM_HINTS(WM_HINTS):
		Client accepts input or input focus: True
		Initial state is Normal State.
		window id # of group leader: 0x4a0000b
WM_CLIENT_MACHINE(STRING) = "RHEL-FX-workstation-01"
_NET_WM_PID(CARDINAL) = 36884
_NET_WM_SYNC_REQUEST_COUNTER(CARDINAL) = 77594681
WM_CLASS(STRING) = "Houdini FX", "Houdini FX"
WM_PROTOCOLS(ATOM): protocols  WM_DELETE_WINDOW, WM_TAKE_FOCUS, _NET_WM_PING, _NET_WM_SYNC_REQUEST
WM_NORMAL_HINTS(WM_SIZE_HINTS):
		user specified location: 21, -25
		user specified size: 1919 by 1080
		program specified minimum size: 901 by 678
		program specified maximum size: 4480 by 1440
		window gravity: Static
```
```


### OS / Distro

6.2.0-36-generic #37~22.04.1-Ubuntu

### Additional System Information (optional)

```text
CPU: AMD Ryzen 9 3950X (32) @ 3.500GHz
GPU: NVIDIA GeForce RTX 3070
```

### LeftWM Check

```text
:: LeftWM version: 0.4.1
:: LeftWM git hash: NONE
:: Enabled features: default journald-log lefthk lefthk-core tracing-journald
:: Checking feature dependencies . . .
    -> journald-log OK
    -> lefthk OK
:: Loading configuration . . .
    -> Configuration loaded OK
:: Checking keybinds . . .
ERROR: Multiple commands bound to key combination modkey + l:
    -> (List(["modkey"]), "l")
    -> FocusWorkspaceNext
Help: change one of the keybindings to something else.

:: Checking environment . . .
    -> Environment OK
:: Checking theme . . .
ERROR: Could not parse theme file: 2:16: Expected option
```

**Comment by Eskaan** [comment]
Could you try if this issue is fixed with 39211ba (#1166)? We just recently merged it and it's not on a release yet.

**Comment by NicTanghe** [comment]
Is it in the 5.1 release ?

**Comment by Eskaan** [comment]
At least some of the issues should be fixed with 5.1, that's probably the width.
I don't think the new release has anything to do with the windows not spawning, but I can't say for certain.

Version bump is a legitimate term to by knowledge :smile:

**Comment by NicTanghe** [comment]
After further use, it seems nothing has changed.

**Comment by NicTanghe** [comment]
It seems the problem is indeed fixed by the 5.1 patch, the windows don`t come back after moving them, but the same behaviour is present in gnome. so the changes did fix evrything
